docs/markdown/releasenotes.md

```diff
@@ -3,24 +3,30 @@
 ## Version 7.10.0
 
 ### Resolved issues
+
 * (IDETECT-2949) Fixed an issue where [solution_name] failed to properly parse Go module version names containing '-' characters.
 * (IDETECT-2959) Fixed an issue where [solution_name] would not fail when running `go mod why` fails.
 * (IDETECT-2971) Fixed an issue where [solution_name] would not produce unique code location paths for Pnpm projects.
+* (IDETECT-2939) Fixed an issue where [solution_name] would not exclude NPM peer or dev dependencies if a project had no declared dependencies.
```

Comment:
Change to read: "(IDETECT-2939) Fixed an issue where [solution_name] projects that had no declared dependencies would not exclude NPM peer or dev dependencies."